### DISCUSSION TITLE
Create online ad maker for Android Java with java

### DISCUSSION THREAD

**Discussion by mdtuhinmia** [community/community#110444] [Programming Help]
### Select Topic Area

Question

### Body

Create online ad maker for Android Java with java

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mdtuhinmia ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion. Adding a few more details may make your post more likely to get a response. 

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by gbpii** [comment]
You can check Android developer guide: https://developer.android.com/media/media3/exoplayer/ad-insertion#java

Adding ads to you app guide by DevTeam.Space: https://www.devteam.space/blog/how-to-add-ads-to-your-app/

You can search for keywords like: "admob" "android firebase ads" "firebase admob" "android java app google ads"
